src/message-dispatch.js

```diff
@@ -78,10 +79,16 @@ export default class MessageDispatch {
         this.scene.emit("quack");
         break;
       case "scene":
-        this.hubChannel.updateScene(args[0]);
+        err = this.hubChannel.updateScene(args[0]);
+        if (err === "unauthorized") {
```

Comment:
ideally the way this would work is:
- every 2-3 minutes, we refresh a perms token and refresh the UI
- if you do not have permission to run these commands, we simply don't include them in the chat commands list

there would still be an edge case if you lost permissions and we hadn't updated perm token yet, but that seems like not a big concern.

Reply:
Arguably, I think we could postpone adding token refresh until we have more discord integration, where we would have to worry about people being added/removed to roles dynamically. Should be easy to do, as you said, but I'm going to punt for now.